core/src/components/item/item.md.scss

```diff
@@ -331,12 +331,12 @@
 // --------------------------------------------------
 
 ::slotted(.button-small) {
-  --padding-top: 0;
-  --padding-bottom: 0;
+  --padding-top: 2px;
+  --padding-bottom: 2px;
   --padding-start: .6em;
   --padding-end: .6em;
 
-  height: 25px;
+  min-height: 25.19px;
```

Comment:
Why the fractional value here?

Reply:
We were seeing some alignment issues but it was reverted: https://github.com/ionic-team/ionic-framework/pull/27547/files#diff-fe00777e1dd0aa7e43d464ac8a996451a125c9ac5644afc60afe7d2535a48a10R339

More info in Liam's comment thread [here](https://github.com/ionic-team/ionic-framework/pull/27547#discussion_r1244378436).